src/base/ContractUtil.ml

```diff
@@ -150,13 +150,13 @@ module ScillaContractUtil (SR : Rep) (ER : Rep) = struct
       ( CUIdentifier.mk_id
           (label_name_of_string MessagePayload.sender_label)
           ER.address_rep,
-        bystrx_typ address_length )
+        Address [] )
```

Comment:
What is the meaning of `Address []`? I don't see that introduced in this PR.

Reply:
Remember that this PR is to be merged into the remote_state_reads branch and not to master. `Address` is a constructor of the `ScillaType.t` datatype.